datasets/senti_lex/senti_lex.py

```diff
@@ -0,0 +1,279 @@
+]
+
+_CITATION = """\
+@inproceedings{inproceedings,
+author = {Chen, Yanqing and Skiena, Steven},
+year = {2014},
+month = {06},
+pages = {383-389},
+title = {Building Sentiment Lexicons for All Major Languages},
+volume = {2},
+journal = {52nd Annual Meeting of the Association for Computational Linguistics, ACL 2014 - Proceedings of the Conference},
+doi = {10.3115/v1/P14-2063}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+This new dataset is designed to solve this great NLP task and is crafted with a lot of care. 
+"""
+
+_HOMEPAGE = "https://sites.google.com/site/datascienceslab/projects/multilingualsentiment"
+
+_LICENSE = "GNU General Public License"
+
+
+class SentiLex(datasets.GeneratorBasedBuilder):
+    """A dataset containing sentiment lexicons for 81 different languages"""
+
+    @property
+    def manual_download_instructions(self):
+        return """\
+    You should download the dataset from https://www.kaggle.com/rtatman/sentiment-lexicons-for-81-languages?select=sentiment-lexicons
+    The webpage requires registration.
+    To unzip the .tar file run `tar -zxvf complete.tar`. To unzip the .gz files
+    run `gunzip train.json.gz` , ...
+    After downloading, please put the files in the sentiment-lexicon folder in a dir of your choice,
+    which will be used as a manual_dir, e.g. `~/.manual_dirs/senti_lex`
+    SentiLex can then be loaded via:
+    `datasets.load_dataset("senti_lex", data_dir="~/.manual_dirs/senti_lex")`.
+    """
+
+    VERSION = datasets.Version("1.1.0")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name=(i[1] + "_words_" + i[0]),
+            version=datasets.Version("1.1.0"),
+            description=("Lexicon of " + i[1] + " words for the " + i[0] + " language"),
+        )
+        for i in zip(LANGS * 2, (["positive"] * len(LANGS)) + (["negative"] * len(LANGS)))
+    ]
+
+    # DEFAULT_CONFIG_NAME = "all"
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "token": datasets.Value("string"),
+                "label": datasets.features.ClassLabel(
+                    names=[
+                        "positive",
+                        "negative",
+                    ]
```

Comment:
```suggestion
                    names=[
                        "negative",
                        "positive",
                    ]
```
usually we put the positive label in second position (index 1)